.bashrc

```diff
@@ -80,8 +82,38 @@ if [ -x /usr/bin/dircolors ]; then
     alias egrep='egrep --color=auto'
 fi
 
+# Initiate virtualenv
 WORKON_HOME="/home/vagrant/.virtualenvs"
 source /usr/local/bin/virtualenvwrapper.sh > /dev/null 2>&1
 cd /vagrant
 
-# the workon command to be added during provisioning 
+# Ask for git configuration data from user based on input
+chmod 755 /home/vagrant/gitconfig
+export PATH=$PATH:/home/vagrant/
+echo 'export PATH=$PATH:/home/vagrant/' >> ~/.profile
```

Comment:
I read this article and find that you don't need to escape the $ as it will in this .bashrc script just add this new directory /home/vagrant/ to the path immediately without waiting for a restart. If we don't do it this way, then .profile will only be active with either a command `source ~/.profile` or waiting for the user to restart the vm. Now if in addition to adding this permanently to ~/.profile, I also run 

`export PATH=$PATH:/home/vagrant/`

inside ~/.bashrc, then it becomes immediately available for the user and using just gitconfig, they can launch the script.